ARCHITECTURE.md

```diff
@@ -106,6 +108,38 @@ This table keeps track of the manifests of the validators.
 | `revoked`            |Whether the manifest has been revoked.                      |
 | `seq`                |The sequence number of this manifest.                       |
 
+### `ballot`
```

Comment:
IMO this should be `amendment_ballot` or something - `ballot` is a bit vague

Reply:
Maybe `validator_voting` since it also has fee votes not just amendments